src/editors/data/constants/requests.js

```diff
@@ -29,4 +29,9 @@ export const RequestKeys = StrictDict({
   uploadImage: 'uploadImage',
   fetchAdvancedSettings: 'fetchAdvancedSettings',
   fetchVideoFeatures: 'fetchVideoFeatures',
+  fetchV1Libraries: 'fetchV1Libraries',
+  fetchV2Libraries: 'fetchV2Libraries',
+  fetchV2LibraryMetadata: 'fetchV2LibraryMetadata',
+  fetchV2LibraryContent: 'fetchV2LibraryContent',
+  fetchBlockContent: 'fetchBlockContent',
```

Comment:
It looks like this api call is not made or used atm, right?

Reply:
`fetchV2LibraryContent` is used but `fetchV2LibraryMetadata` is not. This was originally for fetching v1 library metadata (before I renamed some things to v2). V1 library metadata is needed to get v1 library version. I'll need to put this back in.

Reply:
We acttually need none of:

fetchV2LibraryContent 
fetchV2LibraryContent

as we shouldn't be getting the content from the library.

Reply:
`fetchV2LibraryContent` points to the `/api/libraries/v2/${libraryId}/blocks/` endpoint. We want this for library blocks right? Do I just need to rename this to `fetchV2LibraryBlocks` or is there something else I'm missing?

Reply:
I'm realizing there is more to this than I may have let on. from a UI perspective.
There are three-ish cases we need to consider here:

1. A user clicks edit on a library block which already has a source library_id then either:
- they view their selected blocks in the library.
- they click on a new library.
3. A user clicks  edit on a library block which does not already have a souce library_id.

In theory, in the first case, we need to show the library content block's children (not the library's) to be available for selection.

the last two cases are a conundrum. We know that child blocks of a library and child blocks of a library content block have different ids. Therefore, it is fruitless to attempt to do block selection in those cases. How can we fix this? I think we should do this as a fast-follow, but definitely this is a blocker to rollout.

Reply:
Just checking to see if I understand this correctly... Is the library content block's children the candidates of what might be an older version of the selected library?